plugins/vpp/l3plugin/vppcalls/vpp1908/route_dump_test.go

```diff
@@ -35,13 +35,36 @@ func TestDumpStaticRoutes(t *testing.T) {
 	ifIndexes.Put("if1", &ifaceidx.IfaceMetadata{SwIfIndex: 1})
 	ifIndexes.Put("if2", &ifaceidx.IfaceMetadata{SwIfIndex: 2})
 
-	ctx.MockVpp.MockReply(&ip.IPFibDetails{
-		Path: []ip.FibPath{{SwIfIndex: 2}},
-	})
-	ctx.MockVpp.MockReply(&vpe.ControlPingReply{})
-	ctx.MockVpp.MockReply(&ip.IP6FibDetails{
-		Path: []ip.FibPath{{SwIfIndex: 1}},
-	})
+	ctx.MockVpp.MockReply(&ip.IPRouteDetails{
+		Route: ip.IPRoute{
+			Prefix: ip.Prefix{
+				Address: ip.Address{
+					Af: ip.ADDRESS_IP4,
+					Un: ip.AddressUnionIP4(ip.IP4Address([4]uint8{10, 0, 0, 1})),
```

Comment:
No need for extra convert here, `ip.AddressUnionIP4(ip.IP4Address{10, 0, 0, 1}) suffices here, since `ip.IP4Address` is already defined as 4-byte array.